python/test/hive/util/test_hivesparkddl_to_bq.py

```diff
@@ -0,0 +1,72 @@
+"""
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+"""
+
+import mock
```

Comment:
pyunits are not executing, can you please recheck?

Reply:
<img width="1387" alt="image" src="https://user-images.githubusercontent.com/56369732/203083799-4d940b60-92df-4b50-891e-7626335323bb.png">

Reply:
it was working last week so let me look into why it stopped